transformers/run_eval.py

```diff
@@ -4,47 +4,95 @@
 from transformers import pipeline
 import evaluate
 from normalizer import data_utils
+import time
+from tqdm import tqdm
 
 wer_metric = evaluate.load("wer")
 
 
-def dataset_iterator(dataset):
-    for i, item in enumerate(dataset):
-        yield {**item["audio"], "reference": item["norm_text"]}
-
-
 def main(args):
     asr_pipe = pipeline(
-        "automatic-speech-recognition", model=args.model_id, device=args.device
+        "automatic-speech-recognition", model=args.model_id, device=args.device, batch_size=args.batch_size
     )
 
+    if asr_pipe.model.can_generate():
+        gen_kwargs = {"max_new_tokens": 128}
```

Comment:
Setting to 128 terminates crazy hallucinations earlier, but chops off the end of transcriptions for some short-form datasets (e.g. VoxPopuli and AMI). Probably 256 is more appropriate here.
```suggestion
        gen_kwargs = {"max_new_tokens": 256}
```